src/yarp.c

```diff
@@ -722,7 +722,8 @@ lex_token_type(yp_parser_t *parser) {
         // / /=
         case '/':
           if (match(parser, '=')) return YP_TOKEN_SLASH_EQUAL;
-          if (*parser->current.end == ' ') return YP_TOKEN_SLASH;
+          if (parser->previous.type == YP_TOKEN_INTEGER || parser->previous.type == YP_TOKEN_IDENTIFIER || parser->previous.type == YP_TOKEN_PARENTHESIS_RIGHT)
```

Comment:
I think this check still isn't 100% correct. Can I check that the previous node is a type of `expression` maybe?